test/app/main/api/handlers/newMnemonics.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {newMnemonics} from "app/main/api/handlers"
+import {asRuntimeType} from "app/common/runtimeTypes"
+import {NewMnemonicsResponse} from "app/common/runtimeTypes/storage/wallets"
+
+describe("NewMnemonics Handler", () => {
+
+  it("should return a non-empty mnemonics as response", async () => {
+    const system = {
+      appStateManager: {
+        update: () => undefined
+      }
+    }
+
+    const result = await newMnemonics(system, {})
+
+    // Check that the return value is an unconsolidated wallet
+    const mnemonicsResponse = asRuntimeType(result, NewMnemonicsResponse)
```

Comment:
Comment in line `16` does not match implementation in line `17` (mismatching runtime types).